lib/source/file.js

```diff
@@ -0,0 +1,167 @@
+/* global Log, Config */
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+const fs = require('fs');
+
+class FileParser {
+  constructor() {
+    this.properties = Object.create(null);
+  }
+
+  /**
+   * Parse the file contents
+   * @param {Buffer} data
+   */
+  update(data) {
+    let properties = {};
```

Comment:
Can you elaborate on this? `Object.create(prototype)` is just setting `__proto__` to the `prototype` argument before returning a new instance of `Object` to you.